### ISSUE TITLE
Error occurs in ofx.Ccv.cpp once you clicl 'Add Samples'

### ISSUE THREAD

**Issue by Martinukai** [AndreasRef/doodleTunesNabi#2]
After line 158 of ofx.Ccv.cpp, an error occurs:

line 157: vector<float> ofxCcv::encode(const ofPixels& pix, int layer) const {
line 158: convnet->count = layer;   Thread 1: EXC_BAD_ACCESS (code=1, address=0x18)

Any help on how to solve this would be greatly appreciated. 

Thank you.

**Comment by Martinukai** [comment]
The issue is very similar to one I previously encountered in doodleClassifier in ml4a-ofx that is outlined in this thread:

[https://github.com/ml4a/ml4a-ofx/issues/22](url)

However, the solutions suggested there do not seem to apply to doodleTunesNabi, as the code is quite different.

**Comment by Martinukai** [comment]
After some trial and error, I solved it by first placing the doodleTunesNabi folder inside the apps folder of ml4a-ofx-master. I then copied the image-net-2012.sqlite3 file that I had previously placed within the data folder of ml4a-ofx-master (in accordance with the advice given by @stlou here: [https://github.com/ml4a/ml4a-ofx/issues/22](url)

It worked after that and thankfully it seems to work okay with the ableton file provided too. Thanks so much guys. This project is a big inspiration :) All the best!
